style.css

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Rubik+Gemstones&display=swap');
+*{
+    font-family: 'Rubik Gemstones', cursive;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    min-height: 100vh;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background: #000;
+}
+.container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+.container a{
+    position: relative;
+
+    width: 170px;
+    height: 60px;
+    margin: 20px;
+
+
+    display: inline-block;
+
+    background: #fff;
+}
+.container a::after,
+.container a::before{
+    content: "";
+    position: absolute;
+
+    inset: 0;
+
+    background: red;
+    transition: 0.5s;
+}
+.container a span{
+    position: absolute;
+
+    top: 0;
```

Comment:
Top and left should be along position absolute

Reply:
i just correct as per your suggetions bro...